src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -0,0 +1,70 @@
+package io.iohk.ethereum.sync
+
+import io.iohk.ethereum.FlatSpecBase
+import io.iohk.ethereum.sync.util.RegularSyncItSpecUtils.FakePeer
+import io.iohk.ethereum.sync.util.SyncCommonItSpec._
+import monix.execution.Scheduler
+import org.scalatest.BeforeAndAfter
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.duration._
+
+class RegularSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfter {
+  implicit val testScheduler = Scheduler.fixedPool("test", 16)
+
+  it should "should sync blockchain with same best block" in customTestCaseResourceM(FakePeer.start2FakePeersRes()) {
+    case (peer1, peer2) =>
+      val blockNumer: Int = 2000
+      for {
+        _ <- peer2.importBlocksUntil(blockNumer)(IdentityUpdate)
+        _ <- peer1.connectToPeers(Set(peer2.node))
+        _ <- peer1.startRegularSync().delayExecution(50.milliseconds)
+        _ <- peer2.broadcastBlock()(IdentityUpdate).delayExecution(500.milliseconds)
+        _ <- peer1.waitForRegularSyncLoadLastBlock(blockNumer)
+      } yield {
+        assert(peer1.bl.getBestBlockNumber() == peer2.bl.getBestBlockNumber())
```

Comment:
here and evey where we could probably assert that peers best block hashes are the same

Reply:
Done